integration_tests/src/main/scala/com/nvidia/spark/rapids/tests/tpcds/TpcdsLikeSpark.scala

```diff
@@ -4695,3 +4700,44 @@ object TpcdsLikeSpark {
 }
 
 // scalastyle:on line.size.limit
+
+object ConvertFiles {
+  /**
+   * Main method allows us to submit using spark-submit to perform conversions from CSV to
+   * Parquet or Orc.
+   */
+  def main(arg: Array[String]): Unit = {
+    val conf = new FileConversionConf(arg)
+    BenchUtils.validateCoalesceRepartition(conf.coalesce, conf.repartition)
```

Comment:
Shouldn't this be handled in `FileConversionConf` rather than the caller needing to remember to do this?  There's already a `verify()` call in its constructor to validate other options.